src/settings/LoanHistory/ExceptionCard.js

```diff
@@ -0,0 +1,100 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Field } from 'redux-form';
+import {
+  intlShape,
+  injectIntl,
+  FormattedMessage,
+} from 'react-intl';
+
+import {
+  Col,
+  Row,
+  IconButton,
+  Select,
+} from '@folio/stripes/components';
+
+import RadioButtonSection from '../components/RadioButtons/RadioButtonSection';
+import optionsGenerator from '../utils/options-generator';
+import {
+  closingTypes,
+  closedLoansRules,
+} from '../../constants';
+
+import css from './ExceptionCard.css';
+
+class ExceptionCard extends Component {
+  static propTypes = {
+    intl: intlShape.isRequired,
+    pathToException: PropTypes.string.isRequired,
+    paymentMethods: PropTypes.arrayOf(PropTypes.shape({
+      value: PropTypes.string.isRequired,
+      label: PropTypes.string.isRequired,
+    })).isRequired,
+    exceptionIndex: PropTypes.number.isRequired,
+    onRemoveException: PropTypes.func.isRequired,
+  }
+
+  constructor(props) {
+    super(props);
+
+    this.generateOptions = optionsGenerator.bind(null, props.intl.formatMessage);
+  }
+
+  onRemove = () => {
+    const {
+      exceptionIndex,
+      onRemoveException,
+    } = this.props;
+
+    onRemoveException(exceptionIndex);
+  };
+
+  render() {
+    const {
+      pathToException,
+      paymentMethods,
+    } = this.props;
+    const selectedPayments = this.generateOptions(paymentMethods, 'ui-circulation.settings.loanHistory.selectPaymentMethod');
+
+    return (
+      <Row
+        className={css.exceptionCard}
+        data-test-exception-card
+      >
+        <Col xs={12}>
```

Comment:
Do we really need xs here? Try removing it and check the result.

Reply:
I disagree. The 'xs' attribute specifies the width of the parent element.

Reply:
Sound good.